zmon_worker_monitor/builtins/plugins/time_.py

```diff
@@ -29,15 +29,19 @@ def create(self, factory_ctx):
 
 
 class TimeWrapper(object):
-    def __init__(self, spec='now', utc=False):
+    def __init__(self, spec='now', epoch=None, utc=False):
```

Comment:
why not make it a single argument? using time(123123) looks much simpler than a keyword arg

We could simply assume every int/long is epoch :smile: